### ISSUE TITLE
[BUG] [GraphScope-0.17.0]Load HDFS Data failed Install GraphScope locally without Kubernetes

### ISSUE THREAD

**Issue by JackyYangPassion** [alibaba/GraphScope#2192]
**Describe the bug**
deploy locally without K8S

**To Reproduce**
1. loaded data from HDFS failed
```python
# Install graphscope package if you are NOT in the Playground
!pip3 install graphscope

# Import the graphscope module
import graphscope
import os
from graphscope.client.session import get_default_session
# 单机验证 加载图数据 from HDFS
from graphscope.framework.loader import Loader

graphscope.set_option(show_log=True)  # enable logging

sess = get_default_session()
graph = sess.g()

graph = (
        graph.add_vertices(Loader("hdfs:///home/ogbn_mag_small/paper.csv",host="xxx", port=9000), "paper")
        .add_vertices(Loader("hdfs:///home/ogbn_mag_small/author.csv",host="xxx", port=9000), "author")
        .add_vertices(Loader("hdfs:///home/ogbn_mag_small/institution.csv",host="xxx", port=9000), "institution")
        .add_vertices(Loader("hdfs:///home/ogbn_mag_small/field_of_study.csv",host="xxx", port=9000), "field_of_study")
        .add_edges(
            Loader("hdfs:///home/ogbn_mag_small/author_affiliated_with_institution.csv",host="xxx", port=9000),
            "affiliated",
            src_label="author",
            dst_label="institution",
        )
        .add_edges(
            Loader("hdfs:///home/ogbn_mag_small/paper_has_topic_field_of_study.csv",host="xxx", port=9000),
            "hasTopic",
            src_label="paper",
            dst_label="field_of_study",
        )
        .add_edges(
            Loader("hdfs:///home/ogbn_mag_small/paper_cites_paper.csv",host="xxx", port=9000),
            "cites",
            src_label="paper",
            dst_label="paper",
        )
        .add_edges(
            Loader("hdfs:///home/ogbn_mag_small/author_writes_paper.csv",host="xxx", port=9000),
            "writes",
            src_label="author",
            dst_label="paper",
        )
    )
```

**Expected behavior**
juputer log 

```
extra diagnostics are as follows: Cannot initialize such filesystem for 'hdfs:///home/ogbn_mag_small/paper.csv', exception is:
Traceback (most recent call last):
  File "/home/data/anaconda3/lib/python3.9/site-packages/vineyard/drivers/io/adaptors/read_bytes.py", line 130, in read_bytes
    fs = fsspec.filesystem(protocol, **storage_options)
  File "/home/data/anaconda3/lib/python3.9/site-packages/fsspec/registry.py", line 284, in filesystem
    return cls(**storage_options)
  File "/home/data/anaconda3/lib/python3.9/site-packages/fsspec/spec.py", line 76, in __call__
    obj = super().__call__(*args, **kwargs)
  File "/home/data/anaconda3/lib/python3.9/site-packages/fsspec/implementations/arrow.py", line 235, in __init__
    fs = HadoopFileSystem(
  File "pyarrow/_hdfs.pyx", line 89, in pyarrow._hdfs.HadoopFileSystem.__init__
  File "pyarrow/error.pxi", line 143, in pyarrow.lib.pyarrow_internal_check_status
  File "pyarrow/error.pxi", line 114, in pyarrow.lib.check_status
OSError: HDFS connection failed


/tmp/arrow-apache-arrow-7.0.0/cpp/src/arrow/result.cc:28: ValueOrDie called on an error: NotImplemented: Got HDFS URI but Arrow compiled without HDFS support
```

``

**Screenshots**

**Environment (please complete the following information):**
 - GraphScope version: 0.17.0
 - OS: Linux
 - HDFS version: 3.2.2 

**Additional context**

**Comment by siyuan0322** [comment]
This has been fixed today  https://github.com/alibaba/GraphScope/pull/2175
The nightly image will be built tomorrow.

**Comment by JackyYangPassion** [comment]
How can I quickly apply this patch in my local environment

**Comment by JackyYangPassion** [comment]
> This has been fixed today #2175 The nightly image will be built tomorrow.

or pull the 0.17.0 image tomorrow?

**Comment by sighingnow** [comment]
> or pull the 0.17.0 image tomorrow?

Please do `pip3 install -U graphscope --pre` tomorrow.

**Comment by sighingnow** [comment]
Closing as already fixed by #2175.
